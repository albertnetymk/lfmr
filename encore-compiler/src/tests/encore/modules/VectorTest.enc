#! /usr/bin/env encorec -run

import Vector

class Main
  def main() : void {
    (new Vector<int>).there();
    print "hello"
  }
