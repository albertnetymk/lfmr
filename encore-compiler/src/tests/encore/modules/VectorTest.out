vector reporting for duty!
hello
